backend/dbconf.py

```diff
@@ -10,7 +10,7 @@
 
 engine = create_engine(
     db_url,
-    echo=True,  # connect_args={"check_same_thread": False}
+    echo=True,   connect_args={"check_same_thread": False}
```

Comment:
@muratakburak I think you've removed this? Was there any reason for it? We require this so it'd be nice if we can keep this.